packages/manager/modules/pci/src/projects/new/order-cart.service.js

```diff
@@ -0,0 +1,145 @@
+      .$promise;
+  }
+
+  assignCart(cartId) {
+    return this.OvhApiOrder.Cart().v6()
+      .assign({ cartId })
+      .$promise;
+  }
+
+  deleteCart(cartId) {
+    return this.OvhApiOrder.Cart().v6()
+      .delete({ cartId })
+      .$promise;
+  }
+
+  getProductOffers(cartId, productName) {
+    return this.OvhApiOrder.Cart().Product().v6()
+      .get({
+        cartId,
+        productName,
+      })
+      .$promise;
+  }
+
+  addProductToCart(cartId, productName, product) {
+    return this.OvhApiOrder.Cart().Product().v6()
+      .post({
+        cartId,
+        productName,
+      }, product)
+      .$promise;
+  }
+
+  getProductOptions(cartId, productName, productPlanCode) {
+    return this.OvhApiOrder.Cart().Product().v6()
+      .getOptions({
+        cartId,
+        productName,
+        ...productPlanCode,
+      })
+      .$promise;
+  }
+
+  addProductOptionToCart(cartId, productName, option) {
+    return this.OvhApiOrder.Cart().Product().v6()
+      .postOptions({
+        cartId,
+        productName,
+        ...option,
+      })
+      .$promise;
+  }
+
+  addConfigurationItem(cartId, itemId, label, value) {
+    return this.OvhApiOrder.Cart().Item().Configuration().v6()
+      .post({
+        cartId,
+        itemId,
+      }, {
+        label,
+        value,
+      })
+      .$promise;
+  }
+
+  getCheckoutInformations(cartId) {
+    return this.OvhApiOrder.Cart().v6()
+      .getCheckout({
+        cartId,
+      })
+      .$promise;
+  }
+
+  async checkoutCart(cartId, checkout) {
+    const order = await this.OvhApiOrder.Cart().v6()
+      .checkout({
+        cartId,
+        ...checkout,
+      })
+      .$promise;
+
+    return order;
+  }
+
+  getProductPublicCatalog(ovhSubsidiary, productName) {
+    return this.OvhApiOrder.Catalog().Public().v6()
+      .get({
+        ovhSubsidiary,
+        productName,
+      })
+      .$promise;
+  }
+
+  getProductServiceOptions(productName, serviceName) {
+    return this.OvhApiOrder.CartServiceOption().v6()
+      .get({
+        productName,
+        serviceName,
+      })
+      .$promise;
+  }
+
+  addProductServiceOptionToCart(cartId, productName, serviceName, serviceOption) {
+    return this.OvhApiOrder.CartServiceOption().v6()
+      .post({
+        cartId,
+        productName,
+        serviceName,
+        ...serviceOption,
+      })
+      .$promise;
+  }
+
+  getCartItems(cartId) {
+    return this.OvhApiOrder.Cart().Item().v6()
+      .query({ cartId }).$promise;
+  }
+
+  deleteItem(cartId, itemId) {
+    return this.OvhApiOrder.Cart().Item().v6()
+      .delete({ cartId, itemId }).$promise;
+  }
+
+  deleteAllItems(cartId) {
+    this.OvhApiOrder.Cart().Item().v6().resetQueryCache();
+    return this.getCartItems(cartId)
+      .then(itemsId => this.$q.all(
+        itemsId.map(id => this.deleteItem(cartId, id)),
+      ));
+  }
+
+  getOrderApiSchema() {
+    return this.OvhApiOrder.v6().schema().$promise;
+  }
+}
```

Comment:
Now available with `wucOrderCartService`

Reply:
When I add `@ovh-ux/ng-ovh-web-universe-components` in public-cloud, I get this error:
```
ModuleBuildError: Module build failed (from manager/node_modules/less-loader/dist/cjs.js):

@import 'ovh-ui-kit/packages/oui-color/_variables';
@import 'ovh-ui-kit/packages/oui-switch/_variables';
^
Can't resolve './ovh-ui-kit/packages/oui-switch/_variables.less' in 'manager/packages/manager/modules/web-universe-components/src/v6UiSwitch'
      in manager/packages/manager/modules/web-universe-components/src/v6UiSwitch/v6-ui-switch.less (line 3, column 0)
```
And I think that the place of such a service should not be in `Wuc` :confused: 
- order cart management can also be present in other universe. I agree that `Wuc` can be used everywhere but the web sounds strange in public-cloud ^^
- if I want a service of Wuc, I need the whole module and in this case I am "blocked" due to errors with a component I don't need.